### PR TITLE
Minor fixes from WatchGintama + Quests from Underground Grave of Ritinis

### PR DESCRIPTION
**Pull request by ttgmichael**
Fixes from screenshots on Discord.
https://cdn.discordapp.com/attachments/103512597889835008/109495298656571392/screenshot_20151029_00006.jpg
https://cdn.discordapp.com/attachments/103512597889835008/109494598304272384/screenshot_20151028_00013.jpg
https://cdn.discordapp.com/attachments/103512597889835008/109494643070099456/screenshot_20151029_00002.jpg

Line ETC_20150918_014638 will look like: 

Pawndel <-- (separate line) has been successfully persuaded!
Bring it to the Apsauga Altar.

### CODE REVIEW

File: ETC.tsv [hunk 1]
```diff
@@ -14633,7 +14633,7 @@ ETC_20150918_014634	$Madakia has been absorbed!
 ETC_20150918_014635	You've installed the bomb!
 ETC_20150918_014636	2. Defeat the monsters that are rushing in!
 ETC_20150918_014637	You have succeeded in Palanx trainings that prepare for the actual battles!
-ETC_20150918_014638	You have succeeded in persuasion!{nl}Bring it to the Apsauga Altar
+ETC_20150918_014638	 have been successfully persuaded!{nl}Bring it to the Apsauga Altar
```
Comment (Sunaries) [inline ETC.tsv]: You persuade 1 Pawndel at a time. Have -> has ?

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -7148,7 +7148,7 @@ QUEST_LV_0200_20150714_007151	Thank you so much.{nl}I hope the reinforcement com
 QUEST_LV_0200_20150714_007152	We didn't treat them badly, but they betrayed us.{nl}Dico Thieves stole the supplies from us.{nl}
 QUEST_LV_0200_20150714_007153	{memo X}They must have taken them to Drasa worship ruins, but since I can't leave this place, I guess we all gonna die.{nl}I am sorry, but can you retrieve the supplies that were stolen by the thieves?
 QUEST_LV_0200_20150714_007154	A few decades ago, the thieves attacked the guard camp or stole the supplies secretly.{nl}They were punished for doing so, but I guess they forgot about it.
-QUEST_LV_0200_20150714_007155	They lured us in front of them and stole the objects from behind.{nl}We should set something up, but our chiefs are fighting against Kaliss Knights.{nl}
+QUEST_LV_0200_20150714_007155	They lured us in front of them and stole the goods from behind.{nl}We should set something up, but our chiefs are fighting against Knights of Kaliss. {nl}
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:7151]: against the Knights of Kaliss.

File: ITEM.tsv [hunk 3]
```diff
@@ -5300,15 +5300,15 @@ ITEM_20150714_005299	An orb that holds the aura
...
 ITEM_20150714_005304	{memo X}Truth Eye's broken pieces. It is only available to collect all fragments. 
-ITEM_20150714_005305	Truth Eyes 
+ITEM_20150714_005305	Eyes of Truth
 ITEM_20150714_005306	{memo X}allows you to see the reality of the soul. Let's try to insubstantial monster. 
 ITEM_20150714_005307	Soul Crystal 
 ITEM_20150714_005308	{memo X}wandering pulling force can pull the soul. It is necessary to find a condensed wanted. 
 ITEM_20150714_005309	Left-click to use 
 ITEM_20150714_005310	{memo X}A memo written by Hones about a teacher's said. You can know how to release the device installed at Ritnis catacombs. Cab read 
-ITEM_20150714_005311	Contract Space Magic Stone 
+ITEM_20150714_005311	Spatial Magic Gem of Contract Fragment
```
Comment (Sunaries) [inline ITEM.tsv]: Spatial Demonic Stone Fragment of Contract
Contract's Spatial Demonic Stone Fragment

Spatial Magic Stone Fragment of Contract
Contract's Spatial Magic Stone

Spatial Magic Gem Fragment of Contract
Contract's Spatial Magic Gem

**Comment by Sunaries** [comment]
:+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**